README.md

```diff
@@ -19,74 +19,94 @@ These are related projects that are either already setup and available for use o
 * [Dockstore](https://dockstore.org): our workflow and tool sharing platform
 * [Toil](https://github.com/BD2KGenomics/toil): our workflow engine, these workflows are shared via Dockstore
```

Comment:
Toil is currently not a component in this stack.

Reply:
It's removed. Wonder whether we should mention Broad's FireCloud here? It doesn't use components, but metadata exchange with it is important IMHO. Opinions?